spec/unit/octo_fetcher_spec.rb

```diff
@@ -265,7 +265,7 @@
                             "diff_url"  => "https://github.com/skywinder/changelog_test/pull/21.diff",
                             "patch_url" => "https://github.com/skywinder/changelog_test/pull/21.patch" },
                         "body"           =>
-                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in changelog with #20" }
+                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in change log with #20" }
```

Comment:
The "title" (in this document, too) 

```
"title" => "Merged br (should appear in changelog with  #20)",
```

needs to go back to what it was before.

Reply:
Yes, just saw this.